UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Common/Models/RadioNavigationalWarning/DTO/RadioNavigationalWarnings.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System.ComponentModel;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace UKHO.MaritimeSafetyInformation.Common.Models.RadioNavigationalWarning.DTO
+{
+    public class RadioNavigationalWarnings
+    {
+        [Key]
+        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
+        public int Id { get; set; }
+
+        [DisplayName("Warning Type")]
+        [Required]
+        public int WarningType { get; set; }
+
+        [Required]
+        [StringLength(32, ErrorMessage = "Reference cannot be longer than 32 characters.")]
+        public string Reference { get; set; }
+
+        [DisplayName("Date Time Group")]
+        [Required]
+        public DateTime? DateTimeGroup { get; set; }
```

Comment:
DateTimeGroup is mandatory field and should not be nullable. Also, its contradictory if nullable field has [Required] annotation.
```suggestion
        public DateTime DateTimeGroup { get; set; }
```

Reply:
Thanks for highlighting, done required changes.